src/ui/castlejoysticks.pas

```diff
@@ -175,6 +177,11 @@   TJoysticks = class
       but it may be called in other cases (e.g. "explicit" joystick backend,
       used by Nintendo Switch, may call this at any moment). }
     property OnChange: TNotifyEvent read FOnChange write FOnChange;
+
+    { Called in case a previously initalized joystick had been unplugged unexpectedly. }
```

Comment:
"had been unplugged unexpectedly" -> "has been disconnected".

So using our terminology, and also removing the word "unexpectedly" -- it is always unexpected :)